python/ray/experimental/iter.py

```diff
@@ -527,6 +539,24 @@ def apply_flatten(it):
             self.local_transforms + [apply_flatten],
             name=self.name + ".flatten()")
 
+    def shuffle(self, shuffle_buffer_size: int) -> "LocalIterator[T]":
```

Comment:
Can you please document this?

Reply:
docstring style is shown in https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html